package.json

```diff
@@ -7,7 +7,7 @@
     "indexer"
   ],
   "scripts": {
-    "lint": "prettier --check . && standard",
+    "lint": "prettier --check . && standard && tsc -p .",
```

Comment:
In my understanding, type checking and linting are not the same. To me it's confusing to include type checks in the lint command - I would expect it to only perform linting

Reply:
I see. I was wondering why we introduced `test:types` in one of the other repos. Now I understand 😄 

I'll rework this.

Reply:
https://github.com/filecoin-station/piece-indexer/pull/9